app/models/conviction_search_result.rb

```diff
@@ -5,12 +5,12 @@ class ConvictionSearchResult
   embedded_in :keyPerson
 
   # TODO: Confirm types
-  field :match_result, type: String
-  field :matching_system, type: String
-  field :reference, type: String
-  field :matched_name, type: String
-  field :searched_at, type: DateTime
-  field :confirmed, type: Boolean
-  field :confirmed_at, type: DateTime
-  field :confirmed_by, type: String
+  field :matchResult,     type: String
```

Comment:
There are a number of changes to set field names to match what the current document looks like in the db. However we can keep our more ruby like syntax using field aliases

```ruby
field :matchingSystem, as: :matching_system, type: String
```

<https://docs.mongodb.com/mongoid/master/tutorials/mongoid-documents/#aliasing-fields>

Might help bring the code base to a more familiar ruby footing??

Reply:
Good point. I'd definitely prefer more consistency with Ruby syntax so I'll make the change!